test/tools/jsontests/Common.h

```diff
@@ -0,0 +1,55 @@
+#pragma once
+#include <test/tools/libtesteth/TestHelper.h>
+using namespace std;
+
+namespace dev
+{
+namespace test
+{
+enum class TestExecution
+{
+    REQUIRE_OPTION_ALL,
```

Comment:
enum members shouldn't be all caps, should CamelCase with first letter uppercase https://github.com/ethereum/aleth/blob/master/CODING_STYLE.md#capitalization

Reply:
This enum seems to conflate two concepts together - whether the tests are run with `--all` and whether it's allowed to refill. This makes it too complicated, you should split it into two enums.

On another note, why do you even need to forbid refilling? I wouldn't be so sure that you'll never ever refill legacy tests (for example what if you decide to change the test format in non-backwards-compatible way?) You control anyway what's being merged into the tests repo, so I don't see the point of complicating this.

Reply:
This is so testeth wont accedentially corrupt the LegacyTests
using too enums looks too heavy for just this options.

Reply:
If you corrupt them locally, you can easily check them out of git again.

This is a mess, these options, I don't understand what they mean, without carefully investigating the code. Compare with:

```
enum class RefillOption
{
    Allowed,
	Forbidden
};

enum class AllOption
{
    Required,
	NotRequired
};
```
```
 if (_refill == RefillOption::Forbidden && (Options::get().fillchain || Options::get().filltests))
    fail
```

```
  if (!Options::get().all && (_all == AllOption::Required || test::inArray(c_timeConsumingTestSuites, casename))
     skip;
```